packages/buttons/README.md

```diff
@@ -14,7 +14,9 @@ npm install react react-dom prop-types styled-components @zendeskgarden/react-th
 
 ## Usage
```

Comment:
There's only three usage code snippets in this README.md. Should the rest of the components be added to usages also? For example, `ChevronButton`, `SplitButton`, `Anchor`, etc. I have kept them off for now.

Also, it could feel redundant because the code snippet would be a copy-paste job from the Storybook examples.

@austin94

Reply:
This may make more sense if we included a single code example that showed all of the available components (besides the composite widgets like `ButtonGroup`).

```
<ThemeProvider>
  <div>
    <Button>..</Button>
    <IconButton>...</IconButton>
    etc.
  </div>
</ThemeProvider>
```

Reply:
The user could navigate to "Show Code" for each Story to see a code example. Is the README supposed to serve the purpose of "Show Code"?

I've decided to leave it as is: I only included what was originally in the Styleguidist setup. Maybe we can do a 1 to 1 migration for now?

Reply:
Sounds reasonable. We probably need to have a discussion re: the purpose of a component README (what is it meant to do? what should it not try to do? how should we consistently link to other sources of truth?) and agree on an overall approach.